build.cake

```diff
@@ -5,7 +5,7 @@
 //////////////////////////////////////////////////////////////////////
 
 var target = Argument("target", "Default");
-var configuration = Argument("configuration", "Debug");
+var configuration = Argument("configuration", "Release");
```

Comment:
Cake itself no longer passes in Release by default, so better to set it here to ensure that CI builds don't mistakenly switch to Debug. Charlie ran into this and we have been updating all repos this way.